wgpu/src/image/vector.rs

```diff
@@ -122,6 +122,7 @@ impl Cache {
                     device,
                     encoder,
                 )?;
+                log::warn!("allocating {} {}x{}", id, width, height);
```

Comment:
I think this should be `debug` or `trace` level.

Reply:
ok i changed to debug. i do think warn is useful to see how much allocating, since it should really be never once application is running, but i do see it reallocating if for example i change svg icon / button depending on some state, but it's ok for me to make debug